labcontrol/Experiment.py

```diff
@@ -57,7 +57,16 @@ def add_device(self, device):
         self.devices[device.name] = device
         #self.locks[device.name] = threading.Lock()
 
-    # Lock function that creates
+    """
```

Comment:
This is the way it is run. We create a list of devices in the <Lab>Controller.py file and add a lock for that group of devices. We then repeat this for any other group of devices that can't run concurrently.